src/App.tsx

```diff
@@ -12,155 +12,30 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            to={'/'}
+            className={({ isActive }) =>
+              cn('navbar-item', { 'has-background-grey-lighter': isActive })
+            }
```

Comment:
Better create a separate function, because you have the same logic in the next Link